### PR TITLE
feat: merge client with streamClient

### PR DESCRIPTION
**Pull request by yuanbohan**
## What's changed and what's your intention?

merge client with stream client, bump version to v0.2.0

## Checklist

- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by Anyc66666666 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -108,16 +98,18 @@ err := tbl.AddRow(2, "127.0.0.2", time.Now())
...
 ```go
 resp, err := cli.Write(context.Background(), tbl)
 ```
 
-###### StreamClient Send into GreptimeDB
+###### Stream Write into GreptimeDB
 
 ```go
-err := streamClient.Send(context.Background(), tbl)
+err := cli.StreamWrite(context.Background(), tbl)
```
Comment (nicecui) [inline README.md:110]: 👍

File: README.md [hunk 2]
```diff
@@ -166,16 +158,18 @@ monitors := []Monitor{
 }
 ```
 
-###### Client Create into GreptimeDB
+###### Create into GreptimeDB
 
 ```go
 resp, err := cli.Create(context.Background(), monitors)
```
Comment (nicecui) [inline README.md:164]: `Create` isn't as good as `Write`, it is confused about creating object locally or writing data to server, how about `cli.ORMWrite(...)`?

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+--http-addr 0.0.0.0:4000 \
+--rpc-addr 0.0.0.0:4001 \
+--mysql-addr 0.0.0.0:4002 \
+--postgres-addr 0.0.0.0:4003
+```
+
+## Insert
+
+- [schema](schema/README.md)
+- [tag](tag/README.md)
```
Comment (nicecui) [inline examples/README.md:17]: I consider that `schema` and `tag` are not good titles.
What does `schema` mean and what does `tag` mean?

File: README.md (hunk 2 above)
Comment (yuanbohan) [reply README.md:164]: `ORM` is not a good option. Maybe merge `Create` and `Write` into one `Write` API is a better choice, I will try it after this PR is merged.

File: examples/README.md (hunk 3 above)
Comment (yuanbohan) [reply examples/README.md:17]: Good point, what about:

- Table & Schema
- Struct with tag

File: README.md (hunk 2 above)
Comment (nicecui) [reply README.md:164]: Java's [API](https://github.com/GreptimeTeam/greptimedb-ingester-java/blob/main/ingester-example/src/main/java/io/greptime/StreamWritePOJOsQuickStart.java#L55) is a great example. Maybe we can learn something from it.

File: examples/README.md (hunk 3 above)
Comment (nicecui) [reply examples/README.md:17]: I'm not sure, we use `low-level API` and `ORM API` in the [user documents](https://github.com/GreptimeTeam/docs/pull/803), maybe we need to discuss same names in both user documents and here.

File: README.md (hunk 2 above)
Comment (yuanbohan) [reply README.md:164]: [GORM](https://gorm.io/docs/) is a better package to refer for naming convention.

**Pull request merged**